js/login.js

```diff
@@ -0,0 +1,16 @@
+const pwInput = document.querySelector(".password");
+const toggleBtn = document.querySelector(".toggle_pw");
+
+toggleBtn.addEventListener("click", () => {
+    pwInput.type = pwInput.type === "password" ? "text" : "password";
+    toggleBtn.src = pwInput.type === "password" ? "img/btn_visibility_off.png" : "img/btn_visibility_on.png";
+    checkPassword.src = pwInput.type === "password" ? "img/btn_visibility_off.png" : "img/btn_visibility_on.png";
+});
```

Comment:
💊 제안
변수는 사용하는 곳에서 선언해주세요~ 전연변수로 선언할 이유가 없어보입니다~

```suggestion
toggleBtn.addEventListener("click", () => {
    const pwInput = document.querySelector(".password");
    
    pwInput.type = pwInput.type === "password" ? "text" : "password";
    toggleBtn.src = pwInput.type === "password" ? "img/btn_visibility_off.png" : "img/btn_visibility_on.png";
    checkPassword.src = pwInput.type === "password" ? "img/btn_visibility_off.png" : "img/btn_visibility_on.png";
});
```